instrumental/drivers/tempcontrollers/covesion.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
+"""
+Driver for VISA control of OC2 crystal oven temperature controller
+Created on Tue Sep 16 2017
+
+@author: dodd
+"""
+
+
+from ... import u, Q_ #, visa
+from .. import _get_visa_instrument, ParamSet
+from . import TempController
+#from .. import VisaMixin, SCPI_Facet
+#from ..util import visa_context
+from pyvisa.constants import *
+from pyvisa import ResourceManager
+import time
+from datetime import timedelta
+from sys import stdout
+import codecs
+import numpy as np
+from time import sleep
+#import visa
+#from pint import unit
+#from .. import _get_visa_instrument
+#from .. import InstrumentTypeError
+#from ... import visa
+
+# def _instrument(params):
+#     inst = _get_visa_instrument(params)
+#     # Should add a check of instrument type here. Not sure how to do this now,
+#     # since querying '*IDN?' doesn't work.
+#     return OC(inst)
+_INST_PARAMS = ['OC_visa_address','version']
```

Comment:
Rename `OC_visa_address` to just `visa_address`.

Also, what is `version`? Does it ever make sense to open a temp controller by version? If not, I'd say remove it here.